src/molecule/scenario.py

```diff
@@ -128,13 +128,19 @@ def prune(self):
         ] + self.config.driver.safe_files
         files = util.os_walk(self.ephemeral_directory, "*")
         for f in files:
-            if not any(sf for sf in safe_files if fnmatch.fnmatch(f, sf)):
-                os.remove(f)
+            try:
+                if not any(sf for sf in safe_files if fnmatch.fnmatch(f, sf)):
+                    os.remove(f)
```

Comment:
Only wrap this line with try-except. We shouldn't expect `fnmatch()` to cause `FileNotFoundError`.